README.md

```diff
@@ -28,15 +34,15 @@ What is the purpose of the `alt` attribute? e.g:
 ```
 
 ```text
-Your answer here.
+alt attributes allow users who have special needs understand/view the content. Mostly used on government sites due to ADA requirements.
```

Comment:
:+1: and special needs aside, if for whatever reason an image doesn't load, it will display the contents of `alt` on the page.